docs/core/docker/docker-basics-dotnet-core.md

```diff
@@ -263,9 +275,9 @@ Here are some next steps you can take:
 
 ## Docker Images used in this sample
 
-The following Docker images are used in this sample
+The following Docker image was used in this sample
 
-* [`microsoft/dotnet:2.1-sdk`](https://hub.docker.com/r/microsoft/dotnet)
+* [`mcr.microsoft.com/dotnet/core/sdk:2.1`](https://hub.docker.com/_/microsoft-dotnet-core-sdk/)
```

Comment:
Since the MCR links don't allow to browse the registry and don't take you anywhere, I think we should just drop the link, do you agree @mairaw?

Anyway I don't think the link should go to dockerhub.

```suggestion
- `mcr.microsoft.com/dotnet/core/sdk:2.1`
```

Reply:
Per @Thraka suggestion, I reverted the changes to this doc as it is in the process of being rewritten.